backend-project/small_eod/generic/management/commands/init_data.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+import urllib.request
+from django.core.management import call_command
+from small_eod.letters.factories import LetterFactory
+import tempfile
+
+URL = (
+    "http://cdn.files.jawne.info.pl/"
```

Comment:
Ja zdaję sobie sprawę, że to twój CDN, ale co myślałbyś o skomitowaniu akurat tego pliku do repo? (może też w `django_teryt_tree`)? czy to zły pomysł?